data_safe_haven/commands/typer_init.py

```diff
@@ -1,18 +1,16 @@
 """Command-line application for initialising a Data Safe Haven deployment"""
-# Standard library imports
-from typing import Annotated, Optional
+from typing import Annotated
 
-# Third party imports
 import typer
 
-# Local imports
 from data_safe_haven.functions import validate_aad_guid
+
 from .initialise_command import InitialiseCommand
```

Comment:
Do we want this to be `from data_safe_haven.commands.initialise_command import InitialiseCommand`?

Reply:
I think they are the same, this file is in the `data_safe_haven.commands` module. And we are allowing relative imports of siblings now.